### PR TITLE
Feature/sc 5681 transfer edit profile to new page object structure

### PR DESCRIPTION
**Pull request by kgorner**
# Description
goal: shift selectors and methods of page-objects/editProfile.js to newly created pageObject structure (page objects per page) and delete this file; all test scenarios are still running

## Links to Tickets or other pull requests
https://ticketsystem.schul-cloud.org/browse/SC-5681

## Changes
-added: page-objects/page/AccountPage (page for user settings)
-added: shared_steps/commonKGO-steps.js (methods that are used /can be used from many feautures) > will be renamed later
-deleted: page-objects/editProfile.js
-NavigationTopPage filled with selectors and methods
-formatting of documents (mainly to see in loginPage.js)
- editProfile-steps.js shortened because of having all login methods in shared_steps/commonKGO-steps.js

## Approval for review
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definiton of Done
More and detailed information on the *definition of done* can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

File: page-objects/pages/AccountPage.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        let currentPassword = await driver.$(this.selectors.currentPasswordInp);
+        await currentPassword.setValue(oldPAssword);
+        let passwordField = await driver.$(this.selectors.newPasswordInpt);
+        await passwordField.setValue(newPassword);
+        let passwordControlField = await driver.$(this.selectors.repeatNewPasswordInp);
+        await passwordControlField.setValue(newPassword);
+        await waitHelpers.waitAndClick(this.selectors.submitAccountDataBtn);
+    },
+
+}
```
Comment (Grzesiek80) [inline page-objects/pages/AccountPage.js:23]: add new empty line here

File: page-objects/pages/NavigationTopPage.js [hunk 2]
```diff
@@ -1,11 +1,28 @@
...
+        await waitHelpers.waitAndClick(this.selectors.initialsDDSettings);
+    },
+    clickLogout: async function () {
+        await waitHelpers.waitAndClick(this.selectors.initialsDDLogout);
+    },
+    logout: async function () {
+        this.clickInitials();
+        this.clickLogout();
+    },
 }
```
Comment (Grzesiek80) [inline page-objects/pages/NavigationTopPage.js:28]: add new empty line here

File: shared_steps/commonKGO-steps.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+/*NavigationTopPage*/
+When(/^go to initials$/, async function () {
+	await navigationTopPage.clickInitials();
+});
+
+When(/^go to user settings$/, async function () {
+	await navigationTopPage.clickInitials();
+	await navigationTopPage.clickSettings();
+});
```
Comment (Grzesiek80) [inline shared_steps/commonKGO-steps.js:47]: add new empty line here

File: step_definitions/editProfile-steps.js [hunk 4]
```diff
@@ -1,47 +1,8 @@
...
-When(/^the user (.*) logs in with the new password (.*)$/, function(username, newPassword) {
-	return loginPage.performLogin(username, newPassword)
-});
-
-Then(/^the login must be successful$/, function() {
-	return loginPage.loginResult();
-});
+When(/^change passwort from (.*) to (.*)$/, function (oldPassword, newPassword) {
+	return accountPage.setNewPassword(oldPassword, newPassword);
+});
```
Comment (Grzesiek80) [inline step_definitions/editProfile-steps.js:8]: add new empty line here

File: page-objects/pages/AccountPage.js (hunk 1 above)
Comment (Grzesiek80) [inline page-objects/pages/AccountPage.js:23]: add new empty line here

File: page-objects/pages/NavigationTopPage.js (hunk 2 above)
Comment (kgorner) [reply page-objects/pages/NavigationTopPage.js]: done with commit https://github.com/hpi-schul-cloud/integration-tests/pull/110/commits/0f917ac0ccb5561a4f64ca36af370a68858b6149

File: shared_steps/commonKGO-steps.js (hunk 3 above)
Comment (kgorner) [reply shared_steps/commonKGO-steps.js]: done with commit https://github.com/hpi-schul-cloud/integration-tests/pull/110/commits/0f917ac0ccb5561a4f64ca36af370a68858b6149

File: step_definitions/editProfile-steps.js (hunk 4 above)
Comment (kgorner) [reply step_definitions/editProfile-steps.js]: done with commit https://github.com/hpi-schul-cloud/integration-tests/pull/110/commits/0f917ac0ccb5561a4f64ca36af370a68858b6149

File: page-objects/pages/AccountPage.js (hunk 1 above)
Comment (kgorner) [reply page-objects/pages/AccountPage.js]: done with commit https://github.com/hpi-schul-cloud/integration-tests/pull/110/commits/0f917ac0ccb5561a4f64ca36af370a68858b6149

**Pull request merged**